newIDE/app/src/CommandPalette/CommandPalette.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import ListItemText from '@material-ui/core/ListItemText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+import TextField from '@material-ui/core/TextField';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import CommandsContext from '../CommandPalette/CommandsContext';
+
+type Props = {
+  open: boolean,
+  onClose: () => void,
+};
```

Comment:
Always always use the burger notation in Flow: `{| |}`. Sadly it's not the default in Flow (should be in a future version). Otherwise Flow won't warn if you use put extra props (or make a mistake in a name).
```suggestion
type Props = {|
  open: boolean,
  onClose: () => void,
|};
```